### ISSUE TITLE
Overflow with Table within a Segment

### ISSUE THREAD

**Issue by darewreck54** [Semantic-Org/Semantic-UI-React#2474]
Hi,

I'm not sure if this is a bug or by design.  In the pick below, I put a table within a segment.  However when you resize the browser to a certain size, the table is no longer bounded by the parent segment.
Is this a bug or is there way to achieve the same effect.

Also how would you add overflow scroll bar.  

![image](https://user-images.githubusercontent.com/16171726/35542649-e743ce52-0515-11e8-96f7-64b265b6d2c5.png)

### Steps
1) Go to https://codesandbox.io/s/j4kjox1zzy and observe the overlap
### Expected Result
Expect that as you expand the browser, the table is still bounded within the parent container.  In this case its the segment.   Also when it starts going off the screen, scroll bar appears.  The second part might be based on the caller div to have overflow enabled.

### Actual Result
The table expands past the segment borders.

### Version
0.75.1

### Testcase
https://codesandbox.io/s/j4kjox1zzy

**Comment by brianespinosa** [comment]
@darewreck54 maybe you edited your codepen since you posted this issue... but I cannot duplicate what you are showing in this issue. The screenshot and content in your codepen are both different. At every breakpoint in the viewport, the `Table` does not break out of the `Segment`. I am assuming you edited some of the props to allow the table to collapse, which is how SUI tables are supposed to work.

**Comment by brianespinosa** [comment]
@darewreck54 to add a scrollbar to a `Segment` you would need to add your own CSS. That can be done by compiling your own SUI core CSS, adding your own override styles, or adding inline styles with a `style` prop. That is beyond the scope of this project, but there are people on Stack Overflow that can help you figure that out.

**Comment by darewreck54** [comment]
@brianespinosa Hi brain, Sorry about that.  if you removed the fixed attribute on the table, then the problem will appear.  I've updated it and try it again.  

With that in mind, is this still a invalid issue?

If you don't see it, just keep pasting:

        <Table.HeaderCell>Header</Table.HeaderCell>
   
and press safe and you will see that it start going paste the segment boundary. Also I added the unstackable attribute so you don't see it stacked and see the initial issue.

**Comment by brianespinosa** [comment]
@darewreck54 I see now. Yes, this still is invalid. If you take a look at the core SUI styles and markup, a table can potentially be too wide for a container. It's up to you as the developer to solve this.

As a quick example I saved another version of your codesandbox showing ONLY the `stackable` prop as false on the `Table` to force it never to stack. It breaks out of the `Segment` component this way. You will see I added an inline style to handle the `overflow-x` rule in CSS to make it scroll. This is not a very clean solution because you lose the right padding, but it illustrates how you would want to handle this. I would personally add another `<div>` in between the Segment and Table components with that style instead. That should preserve your padding. Either way, this is a CSS issue, not related to SUIR.

https://codesandbox.io/s/k5807p9zm7

Hope that helps!
